backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/json/JsonPatchMediaTypes.java

```diff
@@ -4,6 +4,10 @@
 
 public class JsonPatchMediaTypes {
 
+	private JsonPatchMediaTypes() {
+		throw new UnsupportedOperationException("This is a utility class and cannot be instantiated");
+	}
```

Comment:
The Java convention is to put methods *after* class fields. This might seem like a nitpick but I feel like this convention is so strongly established that we should follow it everywhere.

Reply:
You're right there, updated in commit [355ada6](https://github.com/DTS-STN/canadian-dental-care-plan/pull/1855/commits/355ada69261da277374adab10e0955da6f3f96bf)